package.json

```diff
@@ -20,7 +20,8 @@
     "stack:docker": "docker run --rm -it -p 8080:8080 -p 5984:5984 -v \"$(pwd)/build\":/data/cozy-app/app cozy/cozy-app-dev",
     "cozyPublish": "git fetch origin ${DEPLOY_BRANCH:-build}:${DEPLOY_BRANCH:-build} && cozy-scripts publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
     "postbuild": "bundlesize",
-    "fixtures": "ACH import fixtures/contacts.json"
+    "fixtures": "ACH import fixtures/contacts.json",
+    "service:checkAndUpdate": "cozy-konnector-dev -m manifest.webapp build/services/checkAndUpdate/contacts.js"
```

Comment:
since we have `services` in the manifest, we should keep `services` and not `service` to not have to think about sing / plu

Reply:
cozy-konnector-dev should be added to the project in devDep ? and run from yarn  cozy-konnector-dev ?

Reply:
cozy-konnector-dev is part of cozy-jobs-cli installed in devDep